README.md

```diff
@@ -24,12 +24,12 @@ dex consists of multiple components:
 	- provides an API for bootstrapping the system.
 - **dexctl** is a CLI tool used to manage a dex deployment
 	- configure identity provider connectors
-	- administer OIDC client identities
+	- administer OIDC client identities.
```

Comment:
@samueldg Okay, just remove the periods from these set of bullet points since they're not complete sentences. Sorry for the delay.